test/functional/legacy/001_script_test_spec.lua

```diff
@@ -0,0 +1,33 @@
+-- First a simple test to check if the test script works.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+local eval = helpers.eval
+
+describe('test script', function()
+  setup(clear)
+
+  it('is working', function()
+    -- Write a single line to test.out to check if testing works at all.
```

Comment:
This comment is confusing, we don't write test.out in the new tests. Suggestion:

    -- Write a single line to check if testing works at all.